HOW_TO.md

```diff
@@ -223,10 +223,10 @@ any template .yaml files.
       6. Save & test
 2. Open grafana dashboard benchmark-runner-report:
    1. Open grafana
-   2. Create(+) -> import -> paste [grafana/benchmark-runner-report.json](grafana/benchmark-runner-report.json) -> Load
+   2. Create(+) -> import -> paste [grafana/benchmark-runner-report.json](grafana/func/benchmark-runner-report.json) -> Load
```

Comment:
Need to fix the link, not just the text.

Reply:
Fixed